Yashodha's org/Customer details API/service.bal

```diff
@@ -0,0 +1,38 @@
+import ballerina/http;
+import ballerinax/java.jdbc;
+import ballerina/io;
+
+# A service representing a network-accessible API
+# bound to port `9090`.
+
+type CustomerDetails record {
+   string customerId;
+   string passportNo;
+   string residentialAddress;
+   string name;
+   string email;
+   string phone;
+ };
+
+service / on new http:Listener(9090) {
+
+    # A resource for generating greetings
+    # + name - the input string name
+    # + return - string name with hello message or error
+    resource function post registerCustomer(string passportNo, string name,string address, string email, string phone) returns string|error? {
+        jdbc:Client jdbcEndpoint = check new ("jdbc:sqlserver://flightreservationsystem.database.windows.net:1433;database=flightdata;user=Yashodha123@flightreservationsystem;password=*******;encrypt=true;trustServerCertificate=false;hostNameInCertificate=*.database.windows.net;loginTimeout=30;");
```

Comment:
It is better to move JDBC configurations into `configuratble` variables and provide the configurations using `Config.toml`.